### PR TITLE
adding Daily data points in measurements

### PR DESCRIPTION
**Pull request by Frankkwizera**

### CODE REVIEW

**Comment by Frankkwizera** [comment]
daily_data_points

File: seshdash/views.py [hunk 1]
```diff
@@ -838,45 +838,66 @@ def graphs(request):
...
+        rmc_account_pk = rmc_account.pk
+        rmc_instance = get_object_or_404(Sesh_RMC_Account, pk = rmc_account_pk)
+        rmc_form = RMCForm(request.POST or None,instance=rmc_instance)
+
+        #checking if the form is valid
+        if form.is_valid() and rmc_form.is_valid():
+            form = form.save()
+            rmc_form = rmc_form.save()
+    else:
+         print "site is function"
```
Comment (jlark) [inline seshdash/views.py]: please remove print statments?

File: seshdash/views.py [hunk 2]
```diff
@@ -838,45 +838,66 @@ def graphs(request):
...
-   sites_stats = get_quick_status(user_sites)
-
-   context_dict['form_edit']= form
-   context_dict['form_add']= form_add
-   context_dict['site_Id']= site_Id
-   context_dict['sites']=sites
-   context_dict['permitted'] = permission
-   context_dict['sites_stats'] = sites_stats
-   return render(request,'seshdash/settings.html', context_dict)
+    context_dict = {}
```
Comment (jlark) [inline seshdash/views.py]: I think this is also the code in your other PR for the edit site functionality?

**Comment by Frankkwizera** [comment]
yeah always forget to remove them.

**Comment by jlark** [comment]
LGTM

**Pull request merged**